packages/jest-resolve/src/__tests__/resolve.test.js

```diff
@@ -78,6 +78,19 @@ describe('findNodeModule', () => {
       paths: (nodePaths || []).concat(['/something']),
     });
   });
+
+  it('it cleans up the path if a query string is present', () => {
```

Comment:
You should not prefix your name with `it`. Also, I think it may benefit from a more explicit statement.

```javascript
it('removes query strings from path before resolving', () => {
```